eventbridge-schedule-to-lambda-cdk-python/README.md

```diff
@@ -0,0 +1,57 @@
+# Amazon EventBridge Schedule to AWS Lambda
+
+This pattern will create an EventBridge schedule to invoke a Python 3.10 based AWS Lambda function every 5 minutes. The pattern is deployed using the AWS CDK for Python. 
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the AWS Pricing page for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one
+  and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS
+  resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [Python 3.10+](https://www.python.org/downloads/) installed
+* [AWS CDK 2.75.0+](https://docs.aws.amazon.com/cdk/latest/guide/cli.html) installed and configured
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd eventbridge-schedule-to-lambda-cdk-python
+    ```
+1. From the command line, bootstrap the AWS CDK if you haven't already:
```

Comment:
missing some instalation for the python library as the dependencies are not found